src/lib/lldpctl.h

```diff
@@ -793,6 +793,7 @@ typedef enum {
 	lldpctl_k_custom_tlv_oui_subtype,	/**< `(I,WO)` custom TLV subtype. Default is 0 (1 byte) */
 	lldpctl_k_custom_tlv_oui_info_string,	/**< `(I,WO)` custom TLV Organizationally Unique Identifier Information String (up to 507 bytes) */
 	lldpctl_k_custom_tlv_op,		/**< `(I,WO)` custom TLV operation */
+	lldpctl_k_config_reset,	/**< `(I)` lldpd is reset */
```

Comment:
This is `(I,WO)`. Also, small nitpick, uses some tabs to align it with the other comments.